### ISSUE TITLE
Do you have plan to support XLM or ROBERTA?

### ISSUE THREAD

**Issue by zongbingwang** [namisan/mt-dnn#149]
Hi, mt-dnn developers, recently, there are some new state of art pretraining model likes ROBERTA and XLM which supports more than 100 languages, do you guys have plan to publish verison 3.0 paper for MT-DNN, and add the feature which supports ROBERTA, XLM or other models which based on hugging face transformers?

And I know that the berf in mt-dnn is also based on hugging face, so it is not hard to implement that, right?

Hope to get your reply, thanks a lot.

**Comment by namisan** [comment]
it supports roberta. 
Pls refer: https://github.com/namisan/mt-dnn/blob/master/experiments/glue/glue_task_def_robert.yml

-xiaodong

**Comment by AnnyTerfect** [comment]
Dear developers, I' m trying to train MT-DNN with ROBERTA encoder these days. I simply changed the '--init_checkpoint' to 'mt_dnn_models/roberta.large', '--task_def' to 'glue_task_def_robert.yml', '--encoder_type' to 2 and kept the other arguments. But I found that the loss hardly decline unfortunately. I would like to ask you whether I need to reset any of the arguments?
